docs/roman/package_index.rst

```diff
@@ -5,12 +5,13 @@ Package Index
    :maxdepth: 2
 
    assign_wcs/index.rst
-   dark_current/index.rst
    datamodels/index.rst
+   dark_current/index.rst
    dq_init/index.rst
-   gain_scale/index.rst
    flatfield/index.rst
+   gain_scale/index.rst
```

Comment:
I suggest we keep these in alphabetical order. So move `dark_current` up.
Why is there a `gain_scale` step? I thought we are not doing this for Roman.